src/main/java/seedu/address/model/person/TagContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
```

Comment:
Is this supposed to be `{@code Tag}` instead of `{@code Name}`? Since filter is applied on tags

Reply:
Yes 😢 Missed this out too..